contracts/bridge/contracts/BasicOutboundChannel.sol

```diff
@@ -18,7 +18,7 @@ contract BasicOutboundChannel is IOutboundChannel {
     /**
      * @dev Sends a message across the channel
      */
-    function submit(address, bytes calldata payload) external override {
+    function submit(bytes calldata payload) external override {
         nonce = nonce + 1;
         emit Message(msg.sender, nonce, payload);
```

Comment:
here, we use the Event as the relay message.
But we don't want to verify the event on the target chain because we must verify the mekle proof of the transaction. Can we use a storage to save this message directly?

Reply:
https://github.com/darwinia-network/darwinia/issues/656#issuecomment-825334641
storage proof should do 2-step proof: account proof and storage proof,
but receipt proof only 1-step proof: receipt proof

Reply:
And storage is more than event.
https://ethereum.stackexchange.com/questions/73777/why-is-storing-via-sstore-more-expensive-than-via-events-mechanism